pocs/camera/camera.py

```diff
@@ -197,10 +197,13 @@ def take_observation(self, observation, headers=None, filename=None, *args, **kw
         exp_time, file_path, image_id, metadata = self._setup_observation(observation,
                                                                           headers,
                                                                           filename,
-                                                                          *args,
                                                                           **kwargs)
 
-        exposure_event = self.take_exposure(seconds=exp_time, filename=file_path, *args, **kwargs)
+        try:
```

Comment:
Much neater syntax, too. As an aside, not entirely sure _why_ I had to start popping the `exp_time` entry from `kwargs`. I started getting errors from the simulator  `take_observation` test (which uses the `exp_time` keyword argument to override the `Observation` `exp_time` attribute) after my refactoring of `take_exposure` because `take_exposure` was being passed multiple values for `seconds`.  The `pop` fixes the problem, but feel that if I understood the root cause better there'd be a more elegant solution.

Reply:
Found and fixed the typo that started the problem, so have removed this `pop` again now.